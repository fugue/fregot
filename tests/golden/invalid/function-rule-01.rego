package p

p(x) = ret {
  ret = "foo"
}

some_test {
  p = 1
}
