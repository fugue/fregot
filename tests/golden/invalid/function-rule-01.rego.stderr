fregot (typecheck error):
  "function-rule-01.rego" (line 8, column 3):
  Unification error:

    8|   p = 1
         ^

  Could not unify type empty with number

  Hint:
    An empty type may mean that you're trying to use a function as a variable
